code/__DEFINES/dcs/signals.dm

```diff
@@ -1284,7 +1284,9 @@
 #define COMSIG_HUMAN_EARLY_UNARMED_ATTACK "human_early_unarmed_attack"
 ///from mob/living/carbon/human/UnarmedAttack(): (atom/target, proximity, modifiers)
 #define COMSIG_HUMAN_MELEE_UNARMED_ATTACK "human_melee_unarmed_attack"
-
+///FROM mob/living/simple_animal/hostile/ooze/eat_atom(): (atom/target, edible_flags)
+#define COMSIG_OOZE_EAT_ATOM "ooze_eat_atom"
+	#define COMSIG_ATOM_EATEN (1<<0)
```

Comment:
this feels more like a proc rather than an actual event, it feels weird.

Reply:
🤷, doesn't feel like the best dcs usage but makes sense.

Reply:
make it COMPONENT_ATOM_EATEN though since all of our response bitflags start with COMPONENT and not COMSIG

Reply:
@AMonkeyThatCodes